.github/workflows/code-quality.yml

```
@@ -59,3 +59,11 @@ jobs:
         uses: github/codeql-action/analyze@v3
         with:
           category: "/language:${{matrix.language}}"
+  
+  fossa-scan:
+      runs-on: ubuntu-latest
```

I'd add a better name for that job. WDYT?
```suggestion
  fossa-scan:
      name: FOSSA Scan
      runs-on: ubuntu-latest
```